src/main/java/com/myjeeva/digitalocean/impl/DigitalOceanClient.java

```diff
@@ -1747,7 +1858,7 @@ private String doDelete(URI uri, HttpEntity entity)
     if (null == entity) {
       HttpDelete delete = new HttpDelete(uri);
       delete.setHeaders(requestHeaders);
-      delete.setHeader(HttpHeaders.CONTENT_TYPE, FORM_URLENCODED_CONTENT_TYPE);
+      delete.setHeader(HttpHeaders.CONTENT_TYPE, JSON_CONTENT_TYPE);
```

Comment:
Previous it used to be `FORM_URLENCODED_CONTENT_TYPE` with DO on delete operation. Just clarifying it, did they updated it JSON?

Reply:
I guess because it doesn't work with `FORM_URLENCODED_CONTENT_TYPE`